Assets/Scripts/Layers/LayerTypes/LayerData.cs

```diff
@@ -175,5 +189,15 @@ public virtual void DestroyLayer()
             ProjectData.Current.RemoveLayer(this);
             LayerDestroyed.Invoke();
         }
+
+        public void AddProperty(LayerPropertyData propertyData)
```

Comment:
Dit is nice, de LayerData is een soort van ingang, als we dan bij het toevoegen van properties in de toekomst ook extra gedrag willen introduceren dan kan dat